scvi/__init__.py

```diff
@@ -2,6 +2,10 @@
 
 """Top-level package for scVI-dev."""
 
-__author__ = """Romain Lopez"""
-__email__ = 'romain_lopez@berkeley.edu'
-__version__ = '0.0.1'
+__author__ = "Romain Lopez, Jeffrey Regier, Maxime Langevin, Edouard Mehlman, Yining Liu"
+__email__ = """Romain Lopez <romain_lopez@berkeley.edu>, 
+            Jeffrey Regier <jregier@eecs.berkeley.edu>, 
+            Maxime Langevin <maxime.langevin@polytechnique.edu>, 
+            Edouard Mehlman <edouard.mehlman@polytechnique.edu>, 
+            Yining Liu <lyining@berkeley.edu>"""
```

Comment:
Do other projects list a lot of authors? If it's more normal to list just one author, I'd just list Romain.

Reply:
It's more common to have one author. I'll change it back.